server/util/body-parser.js

```diff
@@ -110,6 +111,46 @@ export function convertWpList(node) {
   }
 }
 
+export function findWpImageGallery(node) {
+  const className = getAttrVal(node.attrs, 'class');
+  const isWpImageGallery = Boolean(className && className.match('tiled-gallery'));
+
+  if (isWpImageGallery) {
+    try {
+      const images =
+        node.childNodes.filter(
+          r => r.attrs && getAttrVal(r.attrs, 'class') === 'gallery-row'
+        ).map(
+          r => r.childNodes.filter(n => n.attrs && getAttrVal(n.attrs, 'class').match('gallery-group'))
+        ).reduce((acc, group) => acc.concat(group)).map(group => {
+          const img = group.childNodes[0].childNodes[0].childNodes[0];
```

Comment:
I guess this is a safe assumption if they've been created by WordPress the same way all along?